### PR TITLE
add random_sample and aliases

### PR DESCRIPTION
**Pull request by mtar**
## Description

adding random functions and aliases

Issue/s resolved: #553 

## Changes proposed:

- random.random
- random.random_sample
- random.sample
- random.ranf
- random.random_integer

## Type of change

New Feature

## Due Diligence

- [x] All split configurations tested
- [x] Multiple dtypes tested in relevant functions
- [x] Documentation updated (if needed)
- [x] Updated changelog.md under the title "Pending Additions"

#### Does this change modify the behaviour of other functions? If so, which?
no

### CODE REVIEW

File: heat/core/random.py [hunk 1]
```diff
@@ -422,6 +427,43 @@ def randn(*args, dtype=types.float32, split=None, device=None, comm=None):
...
+        Handle to the nodes holding distributed parts or copies of this array.
+    """
+    if not shape:
+        shape = (1,)
+    shape = stride_tricks.sanitize_shape(shape)
+    return rand(*shape, dtype=dtype, split=split, device=device, comm=comm)
+
+
+# aliases
+random_sample = ranf = sample = random
```
Comment (coquelin77) [inline heat/core/random.py:464]: where is `randf` from? i cant find that name in other packages

**Review by coquelin77 — CHANGES_REQUESTED** [review]
looks good to me. only very minor changes before its ready

File: heat/core/random.py (hunk 1 above)
Comment (mtar) [reply heat/core/random.py]: I found `ranf` it as an alias in numpy.

File: heat/core/random.py [hunk 2]
```diff
@@ -422,6 +427,43 @@ def randn(*args, dtype=types.float32, split=None, device=None, comm=None):
...
+    dtype=types.float32,
+    split: Optional[int] = None,
+    device: Optional[str] = None,
+    comm=None,
+):
+    """
+    Random values in a given shape.
+    Create a :class:`~heat.core.dndarray.DNDarray`  of the given shape and populate it with random samples from a
+    uniform distribution over [0, 1).
+    Parameters
```
[the remark this answers is not in the document]
Comment (mtar) [reply heat/core/random.py:442]: fixed

**Review by coquelin77 — APPROVED** [review]

**Review by coquelin77 — APPROVED** [review]

**Pull request merged**